### PR TITLE
Feature/e2e

### PR DESCRIPTION
**Pull request by jseremba**

### CODE REVIEW

File: tests/e2e/LeafletMaps.spec.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import singleLocationPage from "./pages/singleLocation.page";
+import xpathPage from "./pages/xpath.page";
+
+describe("Leaflet maps", () => {
+    const alertValue = "Invalid Coordinates were passed";
+
+    it("should show single location", () => {
```
Comment (McDoyen) [inline tests/e2e/LeafletMaps.spec.ts]: show `a`

File: tests/e2e/LeafletMaps.spec.ts [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        singleLocationPage.open();
+        singleLocationPage.markers.waitForVisible();
+
+        const markerList: WebdriverIO.Element[] = singleLocationPage.markers.value;
+        expect(markerList.length).toBe(1);
+    });
+
+    it("with wrong coodinates should show an alert message", () => {
+        singleLocationPage.open();
```
Comment (McDoyen) [inline tests/e2e/LeafletMaps.spec.ts]: `singleLocationPage` is already open

File: tests/e2e/LeafletMaps.spec.ts [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+        singleLocationPage.longitude.setValue("200");
+        singleLocationPage.longitudeLabel.click();
+
+        singleLocationPage.alert.waitForExist();
+        const alert = singleLocationPage.alert.getText();
+
+        expect (alert).toBe(alertValue);
+    });
+
+    it("when xpath selected should show locations", () => {
```
Comment (McDoyen) [inline tests/e2e/LeafletMaps.spec.ts]: Statment grammar

File: tests/e2e/LeafletMaps.spec.ts (hunk 1 above)
Comment (jseremba) [reply tests/e2e/LeafletMaps.spec.ts]: Fixed

**Pull request merged**